### ISSUE TITLE
setoolkit error

### ISSUE THREAD

**Issue by olly12** [trustedsec/social-engineer-toolkit#486]
am running the newest version of setoolkit i always get stuck at this ''Waiting for payload generation to complete (be patient, takes a bit)... [*] Waiting for payload generation to complete (be patient, takes a bit)... [*] Waiting for payload generation to complete (be patient, takes a bit)... [*] Waiting for payload generation'' pls how can i solve this problem

**Comment by lsoumille** [comment]
Same error here on MacOS High Sierra using the master current version

**Comment by trustedsec** [comment]
Can you paste the options you selected in order to create the payload? Happy to take a peek.

**Comment by lsoumille** [comment]
Social-Engineering Attacks > Infectious Media Generator > File-Format Exploits > Adobe PDF Embedded EXE Social Engineering > Use built-in BLANK PDF for attack > Waiting for payload generation to complete (be patient, takes a bit)... takes infinite time. I'm using MacOS High Sierra. Thanks

**Comment by trustedsec** [comment]
Is the Metasploit location correct? The PDF infectious media generator requires metasploit on that front..

**Comment by lsoumille** [comment]
Yes, The first time I've launched SET, there was an error message saying that the metasploit path was incorrect but I've updated it. So I think it's not related to Metasploit location.
